tools/loadtesting/terraform/r53.tf

```diff
@@ -0,0 +1,99 @@
+resource "aws_route53_zone" "dogfood_fleetctl_com" {
+  name = local.domain_fleetctl
+}
+
+resource "aws_route53_zone" "dogfood_fleetdm_com" {
+  name = local.domain_fleetdm
+}
+
+resource "aws_route53_record" "dogfood_fleetctl_com" {
```

Comment:
Should these be named `fleetctl_com` and `fleetdm_com` instead? or s/dogfood/loadtest/?

Reply:
Uhh, we should probably rethink naming completely at some point now that we have split from the dogfood code.